src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -139,6 +139,27 @@ public function map(callable $callback)
     }
 
     /**
+     * Reload a fresh model instance from the database for all the entities.
+     *
+     * @param  array|string  $with
+     * @return static
+     */
+    public function fresh($with = [])
+    {
+        $model = $this->first();
```

Comment:
Add `$users = new Collection();` in your test before assertion and it will fail with `Error: Call to a member function newQueryWithoutScopes() on null`